src/frontend/js/pages/DashboardOrderLayout/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { Outlet, useParams } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import { DashboardSidebar } from 'widgets/Dashboard/components/DashboardSidebar';
+import {
+  getDashboardRouteLabel,
+  getDashboardRoutePath,
+} from 'widgets/Dashboard/utils/dashboardRoutes';
+import { LearnerDashboardPaths } from 'widgets/Dashboard/utils/learnerRouteMessages';
+import { DashboardLayout } from 'widgets/Dashboard/components/DashboardLayout';
+import { useOrder } from 'hooks/useOrders';
+import { useProduct } from 'hooks/useProduct';
+import { useBreadcrumbsPlaceholders } from 'hooks/useBreadcrumbsPlaceholders';
+import { Product } from 'types/Joanie';
+
+export const DashboardOrderLayout = () => {
+  const params = useParams<{ orderId: string }>();
+  const order = useOrder(params.orderId);
+  const product = useProduct(order.item?.product, { course: order.item?.course });
+  const intl = useIntl();
+  const getRoutePath = getDashboardRoutePath(intl);
+  const getRouteLabel = getDashboardRouteLabel(intl);
+
+  const links = useMemo(
+    () => [
+      {
+        to: getRoutePath(LearnerDashboardPaths.ORDER, { orderId: params.orderId }),
+        label: getRouteLabel(LearnerDashboardPaths.ORDER_RUNS),
```

Comment:
may be the label for `ORDER` is wrong in `LearnerDashboardPaths`

Reply:
Good catch !

Reply:
It is voluntary 😅